bin/flutter

```diff
@@ -15,6 +15,11 @@ SCRIPT_PATH="$FLUTTER_TOOLS_DIR/bin/flutter_tools.dart"
 # TODO(abarth): We shouldn't require dart to be on the user's path.
 DART=dart
 
+if [ -n $FLUTTER_DEV ]; then
+  $DART --checked --package-root="$FLUTTER_TOOLS_DIR/packages" "$SCRIPT_PATH" "$@"
+  exit 0
```

Comment:
Ah, I missed the pub get step. That may be something that we depend on flutter devs running? I run `./dev/update_packages.dart --upgrade` periodically.

We're still in the `set -e` part of the script, so the above line will exit on non-zero codes.